README.md

```diff
@@ -33,3 +31,12 @@ You can transform a spreadsheet using the command line tool by doing:
 ```shell
 ./bin/modsulator spec/fixtures/M1463_AV_manifest.xlsx > output.xml
 ```
+
+## Information about usage for Techies
+- This app does no "writing" and no "saving" -- it JUST does the translation of spreadsheet to MODS
+- Requires all objects to be under the same APO (or it gets ignored?? ... but this code is actually in Argo)
+- users often use Excel spreadsheets for the update
+- It overwrites what is in the datastream;  there is no merging.
+- This currently goes via MODS -- spreadsheet --> MODS --> cocina (via metadata legacy DSA endpoint).
+- Users might call this "replayable spreadsheets"
+NOTE:  this is how it used to work, with Fedora
```

Comment:
Since this doesn't use Fedora anymore, can the Fedora references be removed? (For example, "datastream".)